server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3030;
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const Todo = require('./api/models/todoListModels')
+const cors = require('cors');
```

Comment:
removing this would eliminate a lot of boilerplate and potential sources of confusion for students from this file